canonical-web-and-design/maas-ui
# Fix static paths

Done:
- Set the public url for the built react app.
- Set the correct path for the legacy favicon.
Fixes: #https://github.com/canonical-web-and-design/maas-ui/issues/417.

QA:
- `yarn build-all`
- In ./build/ui/index.html all the hrefs should start with /MAAS/.
- In ./build/legacy/index.html the favicon should start with /MAAS/.
